xml/playercards/FamilyInheritance.xml

```diff
@@ -0,0 +1,37 @@
+<Defaults>
+  <Button padding="30 30 30 30"
+    font="font_teutonic-arkham"
+    textColor = "white"
```

Comment:
Extra whitespace (also multiple times below)

Reply:
I'm not seeing it for some reason?